include/swift/AST/DiagnosticsSema.def

```diff
@@ -6655,6 +6655,9 @@ ERROR(availability_variadic_type_only_version_newer, none,
 NOTE(availability_guard_with_version_check, none,
      "add 'if #available' version check", ())
 
+NOTE(availability_update_attribute, none,
+     "update @available attribute for %0 from %1 to %2 to meet the requirements of '%3'", (StringRef, StringRef, StringRef, StringRef))
```

Comment:
Nit: Should we use quotes on versions as well?
```suggestion
     "update @available attribute for %0 from '%1' to '%2' to meet the requirements of '%3'", (StringRef, StringRef, StringRef, StringRef))
```

Reply:
Done